File: fixtures/dart_output/get_with_all_numerical_data.dart
import 'package:http/http.dart' as http;

void main() async {
  var headers = {
    'Content-Type': 'application/json',
    'Accept': 'application/json',
  };

  var data = '18233982904';

  var res = await http.post('http://198.30.191.00:8309/CurlToNode', headers: headers, body: data);
  if (res.statusCode != 200) throw Exception('post error: statusCode= ${res.statusCode}');
  print(res.body);
}

File: fixtures/node-request/get_with_all_numerical_data.js
var request = require('request');

var headers = {
    'Content-Type': 'application/json',
    'Accept': 'application/json'
};

var dataString = '18233982904';

var options = {
    url: 'http://198.30.191.00:8309/CurlToNode',
    method: 'POST',
    headers: headers,
    body: dataString
};

function callback(error, response, body) {
    if (!error && response.statusCode == 200) {
        console.log(body);
    }
}

request(options, callback);
